flink-streaming-connectors/flink-connector-filesystem/src/test/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSinkFaultToleranceITCase.java

```diff
@@ -69,6 +69,9 @@
 
 	private static String outPath;
 
+	private final String pendingSuffix = ".pending";
```

Comment:
by conventions these should be static; this makes it more obvious to readers that they really never change.